### PR TITLE
Add crt-guest-advanced, update other crt-guest, update ntsc-adaptive

### PR DESCRIPTION
**Pull request by hunterk**

### CODE REVIEW

**Comment by Tatsuya79** [comment]
Merging fields for 300+ width disables artifacting.
So now, the composite preset won't show any rainbow effect on Sonic waterfall.

**Comment by hizzlekizzle** [comment]
better? Thanks for testing, btw.

**Comment by Tatsuya79** [comment]
Yes, that's a better compromise I think. :+1:

File: presets/crt-guest-advanced-ntsc.slangp [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+shader5 = ../ntsc/shaders/ntsc-adaptive/ntsc-pass2.slang
+
+filter_linear4 = false
+filter_linear5 = true
+
+scale_type_x4 = source
+scale_type_y4 = source
+scale_x4 = 4.0
+scale_y4 = 1.0
+frame_count_mod4 = 2
```
Comment (Tatsuya79) [inline presets/crt-guest-advanced-ntsc.slangp:48]: Is this frame_count_mod line necessary to add into every presets using ntsc-adapt now?

File: ntsc/shaders/ntsc-adaptive/ntsc-pass1.slang [hunk 2]
```diff
@@ -81,13 +97,34 @@ vec3 rgb2yiq(vec3 col)
    return col * yiq_mat;
 }
 
+vec3 declip(vec3 c, float b)
```
Comment (Tatsuya79) [inline ntsc/shaders/ntsc-adaptive/ntsc-pass1.slang:100]: This isn't used?

File: presets/crt-guest-advanced-ntsc.slangp (hunk 1 above)
Comment (hizzlekizzle) [reply presets/crt-guest-advanced-ntsc.slangp:48]: No, it's already mod()ed in the shader, so it's extraneous.

**Pull request merged**